src/licenses/entities/license/activeSeatsFindResult.ts

```diff
@@ -3,56 +3,35 @@ import {
   FiltersParameters,
   SortParameters,
 } from '../../licensesClient';
-import { AbstractEntity } from '../../../abstractEntity';
+import {
+  AbstractActiveSeats,
+  ActiveSeatsData,
+  ActiveSeatsFields,
+} from '../../../common/abstract/abstractActiveSeats';
 
-export enum ActiveSeatsFindResultFields {
```

Comment:
This enum has disappeared 
Can you keep it for backward-compatibility?

Reply:
Done